src/System.Security.Cryptography.Pkcs/tests/System.Security.Cryptography.Pkcs.Tests.csproj

```diff
@@ -2,7 +2,12 @@
   <PropertyGroup>
     <ProjectGuid>{2DD8DFFA-09FF-46C6-8313-4A9CC1849A44}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Configurations>netcoreapp-Debug;netcoreapp-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netfx-Windows_NT-Debug;netfx-Windows_NT-Release;netstandard-Debug;netstandard-Release</Configurations>
+    <!--
+      UAP fails when loading the src assembly:
```

Comment:
Are you going to follow up on this issue?

Reply:
I don't plan to. I'm relatively sure we will soon change how UAP is composed in corefx.